chat.delta.desktop.yml

```diff
@@ -102,46 +130,52 @@ modules:
       # use ELECTRON_CACHE to point to them.
       # Taken from https://github.com/electron/chromedriver/issues/28#issuecomment-42783936
       - type: file
-        url: https://github.com/electron/electron/releases/download/v3.0.0/chromedriver-v3.0.0-linux-x64.zip
-        sha256: 31c59d42d80001721dbdbf2f2c8b3b527bdb8df38c766b020329cb46cf1d3bb4
-        dest: npm-cache
+        only-arches: ["x86_64"]
+        url: https://github.com/electron/electron/releases/download/v4.0.0/chromedriver-v4.0.0-linux-x64.zip
+        sha256: 221db18ea2f3a8267f9a57ee61a9d5a6df09824c61bf8cefa360502cae503d7c
+        dest: flatpak-node/electron-cache
+
       - type: file
-        url: https://github.com/electron/electron/releases/download/v3.0.0/SHASUMS256.txt
-        sha256: 6560dd88350be568ff29cdb1cdb05e68304268b821b1d5040f9a5fde901ef670
-        dest-filename: SHASUMS256.txt-3.0.0
-        dest: npm-cache
+        url: https://github.com/electron/electron/releases/download/v4.0.0/SHASUMS256.txt
+        sha256: 52dfb903bf248db3b2c94cdcddd310034cd743098c13b5b10c71272ad6f812d1
+        dest-filename: SHASUMS256.txt-4.0.0
+        dest: flatpak-node/electron-cache
 
       - type: archive
-        url: https://github.com/deltachat/deltachat-desktop/archive/v0.102.0.tar.gz
-        sha256: 7ada40529cc998d37c7ea16e0fac1b5bd5b562ceb4099ffe9769481e15a92f4f
+        url: https://github.com/deltachat/deltachat-desktop/archive/v0.840.0.tar.gz
+        sha256: 16ac2708d1a8728217e1672124e6b94898ebf8c02728b3563ace702f993ad98d
 
       - type: file
         url: https://raw.githubusercontent.com/deltachat/interface/970c5df39866695b5e8ebdd73caa68cbb6de5242/icons/delta-v7-pathed.svg
         sha256: 1787a9dfdd80301ae5dbe48105bd58215c6552bd8ecfa4efd15fe6d31ba31274
         dest-filename: delta-v7-pathed.svg
 
-      - type: file
-        path: package-lock.json
-        dest-filename: package-lock.json
-
       - type: file
         path: chat.delta.desktop.desktop
 
       - type: file
         path: chat.delta.desktop.256.png
         dest-filename: chat.delta.desktop.png
 
-      # We overwrite upstream's appdata file because it lacks the releases
-      - type: file
-        path: chat.delta.desktop.appdata.xml
-        dest-filename: static/chat.delta.desktop.appdata.xml
-
       - type: script
         dest-filename: run.sh
         commands:
           - /app/delta/deltachat-desktop
 
       # All the generated npm downloads, see README.md how to create
-      # this file.
-      # This needs to go to the bottom so that the commands in that file can see package.json and friends.
-      - generated-sources.json
+      # this file.  This needs to go to the bottom so that the
+      # commands in that file can see package.json and friends.
+      - generated-sources-npm.0.json
+      - generated-sources-npm.1.json
+      - generated-sources-npm.2.json
```

Comment:
How come there's 3 of these now?  The readme doesn't explain that part.

Reply:
I've added a paragraph to the readme. Thanks for catching this!